README.md

```diff
@@ -82,14 +82,16 @@ or
 
 The following are the functions provided by `ts-fold-mode`
 
-| Commands                   | Description                                                                 |
-| -------------------------- | --------------------------------------------------------------------------- |
-| `ts-fold-close`            | fold the current syntax node.                                               |
-| `ts-fold-open`             | open all folds inside the current syntax node.                              |
-| `ts-fold-open-recursively` | open the outmost fold of the current syntax node. Keep the sub-folds close. |
-| `ts-fold-close-all`        | close all foldable syntax nodes in the current buffer.                      |
-| `ts-fold-open-all`         | open all folded syntax nodes in the current buffer.                         |
-| `ts-fold-toggle`           | toggle the syntax node at `point'.                                          |
+| Commands                   | Description                                                                                         |
+| -------------------------- | --------------------------------------------------------------------------------------------------- |
+| `ts-fold-mode`             | enable `ts-fold-mode` in the current buffer.                                                        |
+| `global-ts-fold-mode`      | enable `ts-fold-mode` whenever tree-sitter is turned on and the major mode is supported by ts-fold. |
```

Comment:
My thinking was to list them here since they complete the set of functions provided by `ts-fold` as stated on the line above the table. It also communicates that there's a global mode and a local mode for enabling the package. 

The other way this can be structured (which I'm starting to like now that I think about it and will try to add today) is by splitting into two tables: one for commands that enable `ts-fold` and one for the commands within `ts-fold`. In that first table, we can also then add the functions for enabling fold indicator mode with a link to the documentation in the plugin section. 

I think it would be good to add the mention of indicators mode here. When I was first trying out the package, I don't read the full documentation since I wanted to just get it enabled. So I didn't actually know indicators mode existed until I was looking for how to add new folding definitions 😆. All that is to say, since I think indicators mode is awsome

Reply:
Got it! Split into two tables make sense! Thank you! ;)